packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -651,7 +651,8 @@ export function toTask(taskDto: TaskDto): theia.Task {
         result.execution = getProcessExecution(taskDto as ProcessTaskDto);
```

Comment:
just wondering: do we need to use `types.ProcessExecution.is(execution)` in the if-statement in line 650, since you added a "customType" ?

Reply:
thank you, I'll review it today

Reply:
@elaihau I updated the PR, please review.
about your question  - we [use](https://github.com/theia-ide/theia/blob/65ffea53952a98b72ecfacce6daba239ea7e1eeb/packages/plugin-ext/src/plugin/type-converters.ts#L618) `types.ProcessExecution.is(execution)` when we do converting from `theia.Task` to `TaskDto`, but we don't need the same when we converting from  `TaskDto` to `theia.Task`. For this case we use check by type only. 
It's because at converting  `theia.Task` of 'process' type 'process' filed is converted to 'command' filed of `TaskDto`. ProcessTaskRunner [recognizes](https://github.com/theia-ide/theia/blob/65ffea53952a98b72ecfacce6daba239ea7e1eeb/packages/task/src/node/process/process-task-runner.ts#L72) task by type and expects [command field](https://github.com/theia-ide/theia/blob/65ffea53952a98b72ecfacce6daba239ea7e1eeb/packages/task/src/node/process/process-task-runner.ts#L75) at the moment.